File: 2020_Pascal/aoc.pas
program aoc;

uses day1;

const
	file_name = 'day1.txt';

var
	f : TextFile;
	result : longint;
	is_part_b : boolean;

begin
	assign(f, file_name);
	reset(f);

	is_part_b := true;
	result := compute_result(f, is_part_b);
	writeln(result)
end.

File: 2022_JavaScript/aoc.js
import {compute_result} from "./day1.js"

import * as fs from 'fs';

fs.readFile('day1.txt', (err, inputD) =>
{
	if (err) throw err;

	let is_part_b = true;
	var result = compute_result(inputD.toString(), is_part_b);

	console.log(result);
 })